.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["es2015", {"loose": true}]],
-  "plugins": ["transform-object-assign"]
+  "presets": ["es3", ["es2015", {"loose": true}]],
+  "plugins": []
```

Comment:
I don't think the es3 preset includes it. If `Object.assign` is used anywhere, we still need it.